c_test/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+project (c_test)
```

Comment:
Consider moving some of these projects to an example folder so the root is not cluttered

Reply:
let's do this in another PR

---
In reply to: [192497477](https://github.com/JohnLangford/vowpal_wabbit/pull/1492#discussion_r192497477) [](ancestors = 192497477)